app/src/main/java/ch/epfl/sweng/erpa/MainActivity.java

```diff
@@ -1,7 +1,9 @@
 package ch.epfl.sweng.erpa;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
 
 public class MainActivity extends AppCompatActivity {
```

Comment:
Can you please use android.app.Activity instead of android.support.v7.app.AppCompatActivity?